pages/index.vue

```diff
@@ -6,7 +6,12 @@
       </h2>
       <a class="a-link" href="" />
     </div>
-    <Grid />
+    <grid placement="home_page.horizontal_recs1" :rows="1" />
```

Comment:
Can I remove the extra cards or specify other extra cards, because today the extra cards is duplicated

Reply:
@benjick

Reply:
@coopjakob oh yeah, I had some thoughts about this. Do you wanna do something with like the include/exclude fields that you already made? Or should we have one stack of extracards per grid?

Reply:
@benjick We don't have any extra cards in this placement today, so I just want to be able to remove them for now...

Reply:
@coopjakob okay, I'll make it happen 👍